server/notebooks/views.py

```diff
@@ -29,7 +29,9 @@ def _get_iframe_src():
 
 @ensure_csrf_cookie
 def notebook_view(request, pk):
+
     notebook = get_object_or_404(Notebook, pk=pk)
+
```

Comment:
Looks like we added some newlines to this file by mistake.

Reply:
hmm, there was a story of another line of code here once upon a time, but that story is lost to the world. alas